UOP1_Project/Assets/Scripts/Inventory/Item.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+// Created with collaboration from:
+// https://forum.unity.com/threads/inventory-system.980646/
+public abstract class Item : ScriptableObject
+{
+	[Tooltip("The Item's name")]
+	[SerializeField] private string _name; // TODO: Consider using a string table for localization
+	[Tooltip("A preview Image for the Item")]
+	[SerializeField] private Sprite _image;
+	[Tooltip("A prefab reference for the model of the item.")]
+	[SerializeField] private MeshRenderer _model; // use this as reference prefab
```

Comment:
Careful here, since if an item is made of multiple renderers with a common empty GameObject, this might lead to errors or having to double check if the referenced `MeshRenderer` is on the base of the Prefab.
Can't it just simply be a `GameObject` reference?

Reply:
Done. changed field/

@trevorjmcdonald  please Pull https://github.com/trevorjmcdonald/open-project-1/pull/8/commits/abf170dbb9929c939f0c61489275752201fe83f3

pr #8 in your fork